detox/src/devices/runtime/RuntimeDevice.test.js

```diff
@@ -969,6 +969,12 @@ describe('Device', () => {
     expect(device.deviceDriver.captureViewHierarchy).toHaveBeenCalledWith('capture');
   });
 
+  it('getViewHierarchy() should return the result of the driver', async () => {
+    const device = await aValidDevice();
+    const hierarchy = await device.getViewHierarchyXml();
```

Comment:
Make sure you check the result:

```suggestion
    driverMock.driver.getViewHierarchyXml.mockResolvedValue('xml');

    const hierarchy = await device.getViewHierarchyXml();
    expect(hierarchy).toEqual('xml');
```